react-native/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../tslint.json",
+    "rules": {
+        // This package uses the Function type for good reason and `object` is TypeScript 2.2 only.
+        "forbidden-types": false
```

Comment:
Good point re those `Object` cases, I’ll take another look to see if there are places that can be resolved by other more specific types.